core/services/streams/stream_registry.go

```diff
@@ -0,0 +1,64 @@
+package streams
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/chainlink/v2/core/services/pipeline"
+)
+
+type StreamID string
+
+func (s StreamID) String() string {
+	return string(s)
+}
+
+type StreamRegistry interface {
```

Comment:
How about just `streams.Registry`?
```suggestion
type Registry interface {
```